### PR TITLE
Fix: split_independent_vrp respects skills and keeps relations

### PR DESCRIPTION
**Pull request by senhalil**

### CODE REVIEW

File: test/wrapper_test.rb [hunk 1]
```diff
@@ -2883,31 +2883,87 @@ def test_check_distances_if_distance_and_time_with_lateness
                  "all should be infeasible due to distance constraint"
   end
 
-  def test_number_of_service_vrps_generated_in_split_independent
+  def test_split_independent_vrp_does_not_reset_relations
+    problem = VRP.independent_skills
+    # The services in the same realtion would stay on the same vehicle after the split_independent_vrp even without
```
Comment (braktar) [inline test/wrapper_test.rb:2888]:
```suggestion
    # The services in the same relation would stay on the same vehicle after the split_independent_vrp even without
```

File: test/wrapper_test.rb [hunk 2]
```diff
@@ -2883,31 +2883,87 @@ def test_check_distances_if_distance_and_time_with_lateness
...
+    # the relations (due to skills). Here we test only if the relations stay intact after split_independent_vrp.
+    # split_independent_vrp does not take into account relations but if the services are not feasible in the first place
```
Comment (braktar) [inline test/wrapper_test.rb:2890]: Is it possible to ease the understandability of this phrase by reducing the number negations ?
```suggestion
    # split_independent_vrp only takes into account relations if the services are feasible in the first place
```

File: test/wrapper_test.rb [hunk 3]
```diff
@@ -2883,31 +2883,87 @@ def test_check_distances_if_distance_and_time_with_lateness
...
+    # i.e., both cannot be served by the same vehicle, then split cannot do anything about it.
```
Comment (braktar) [inline test/wrapper_test.rb:2891]:
```suggestion
    # i.e., both services of the same relation cannot be served by the same vehicle, then split cannot do anything about it.
```

File: test/wrapper_test.rb (hunk 2 above)
Comment (senhalil) [reply test/wrapper_test.rb:2890]: In fact split_independent_vrp never takes into account relation. I improved the explanation. Unresolve this conversation if it is still not clear.

**Review by braktar — APPROVED** [review]

**Pull request merged**